tests/virt_autotest/reboot_and_wait_up.pm

```diff
@@ -59,7 +58,16 @@ sub reboot_and_wait_up {
             #login is required when sol console is used for the first time
             unless (check_screen('text-logged-in-root')) {
                 #The timeout can't be too small since autoyast installation
-                assert_screen "text-login", 600;
+                #Xen console may output additional messages about vm on sol whose output is disrupted.
+                #So in order to get login prompt back on screen, 'ret' key should be fired up. But the
+                #os name banner might be available anymore, only 'linux-login' needle can be matched.
```

Comment:
missing 'not' after 'might' in the comment?

Reply:
Updated.